cmd/kubectl-testkube/commands/common/helper.go

```diff
@@ -162,6 +162,30 @@ func updateHelmRepo(helmPath string, dryRun bool, isOnPrem bool) *CLIError {
 	return nil
 }
 
+// It cleans existing migrations job with long TTL
+func CleanExistingCompletedMigrationJobs(namespace string) (cliErr *CLIError) {
+	kubectlPath, cliErr := lookupKubectlPath()
+	if cliErr != nil {
+		return cliErr
+	}
+
+	// Clean the job only when it's found and it's state is successful - ignore pending migrations.
+	succeeded, _ := runKubectlCommand(kubectlPath, []string{"get", "job", "testkube-enterprise-api-migrations", "-n", namespace, "-o", "jsonpath={.status.succeeded}"})
```

Comment:
is it okay to skip error or it will give us additional info?

Reply:
I think I would need to filter errors related to state when there is no migration job at all 
Assuming happy path for now